.github/workflows/publish.yml

```diff
@@ -0,0 +1,44 @@
+name: Publish
+on:
+  push:
+    tags:
+      - '*'
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v3
+
+    - name: Set up Python
+      uses: actions/setup-python@v3
+      with:
+        python-version: 3.11
```

Comment:
I would be consistent here with the version of python.  Looks like you use 3.9 elsewhere, so I would 3.9 here.